src/components/Calculator.js

```diff
@@ -0,0 +1,103 @@
+import React, { Component } from 'react';
+
+import Input from './Input';
+import Button from './Button';
+
+class Calculator extends Component {
+    constructor(props) {
+        super(props);
+
+        this.onCurrentNumber1Change = this.onCurrentNumber1Change.bind(this);
+        this.onCurrentNumber2Change = this.onCurrentNumber2Change.bind(this);
+        this.onPlus = this.onPlus.bind(this);
+        this.onMinus = this.onMinus.bind(this);
+        this.onMultiply = this.onMultiply.bind(this);
+        this.onDivide = this.onDivide.bind(this);
+        this.onReset = this.onReset.bind(this);
+
+        this.state = {
+            currentNumber1Value: '',
+            currentNumber2Value: '',
+            result: 0
+        }
+    };
+
+    onCurrentNumber1Change(e) {
+        this.setState({ currentNumber1Value: e.target.value })
+        console.log(this.state.currentNumber1Value);
```

Comment:
you should remove all console.logs, as we don't want to have them in ready code.